src/components/Common/Banner.js

```diff
@@ -1,26 +1,68 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prop-types */
 import { Box } from '@mui/system';
 import { Typography } from '@mui/material';
 import PropTypes from 'prop-types';
+import { useLocation } from 'react-router-dom';
+import { useContext } from 'react';
+import { DatastoreContext } from '../../context/DatastoreProvider';
+import ComparisonSelector from '../Chart/ComparisonSelector';
 import theme from '../../assets/styles/AppTheme';
-import MeasurementYearSelector from './MeasurmentYearSelector';
+import MeasurementYearSelector from '../Chart/MeasurementYearSelector';
+
+function Banner({
+  headerText, lastUpdated, handleResetData, activeMeasure, setIsLoading,
+}) {
+  const { pathname } = useLocation();
+  const {
+    datastore,
+    datastoreActions,
+  } = useContext(DatastoreContext);
+
+  const isRoot = pathname === '/';
+  const isMeasure = /^\/[^/]+$/.test(pathname);
+  const showComparison = !isRoot || isMeasure;
 
-function Banner({ headerText, lastUpdated }) {
   return (
     <Box className="banner">
       <Box className="banner__header-container">
-        <Typography variant="h1" color={theme.palette?.bluegray.D2} className="banner__header">
+        <Typography
+          variant="h1"
+          color={theme.palette?.bluegray.D2}
+          className="banner__header"
+        >
           {headerText}
         </Typography>
-        <Box className="banner__year-selector">
-          <MeasurementYearSelector />
-        </Box>
+
+        {showComparison && (
+          <Box className="banner__comparison-selector">
+            <ComparisonSelector
+              activeMeasure={activeMeasure}
+              handleResetData={handleResetData}
+              setIsLoading={setIsLoading}
+            />
+          </Box>
+        )}
+
+        {!showComparison && (
+          <Box className="banner__year-selector">
+            <MeasurementYearSelector />
+          </Box>
+        )}
```

Comment:
I'm not a fan of removing this, but after some testing I understand why you did. Can we disable it instead?

Reply:
For now, yes. Figured it could be further developed to be compatible... eventually.

Reply:
Makes sense to me.